openhands/server/session/conversation_init_data.py

```diff
@@ -17,6 +18,7 @@ class ConversationInitData(Settings):
     selected_branch: str | None = Field(default=None)
     conversation_instructions: str | None = Field(default=None)
     git_provider: ProviderType | None = Field(default=None)
+    mcp_shttp_servers: list[MCPSHTTPServerConfig] | None = Field(default=None)
```

Comment:
We already have `mcp_config` and `shttp_servers` is just `mcp_config.shttp_servers`. Can we just re-use `mcp_config` to deduplicate stuff?
https://github.com/All-Hands-AI/OpenHands/blob/9b0a5da839256ca32076509a98c30d9854ab55ff/openhands/storage/data_models/settings.py#L42